conf/openmetadata.yaml

```diff
@@ -433,9 +435,14 @@ web:
     option: ${WEB_CONF_PERMISSION_POLICY_OPTION:-""}
   cache-control: ${WEB_CONF_CACHE_CONTROL:-""}
   pragma: ${WEB_CONF_PRAGMA:-""}
-  
+
 operationalConfig:
   enable: ${OPERATIONAL_CONFIG_ENABLED:-true}
   operationsConfigFile: ${OPERATIONAL_CONFIG_FILE:-"./conf/operations.yaml"}  
+
+mcpConfiguration:
+  enabled: ${MCP_ENABLED:-true}
+  path: ${MCP_PATH:-"/mcp"}
+  mcpServerVersion: ${MCP_SERVER_VERSION:-"1.0.0"}  
     
```

Comment:
Should that be its own config file toi seperate server and mcp config (the same way we have operations.yaml)?